node/config.go

```diff
@@ -41,6 +41,9 @@ const (
 	datadirStaticNodes     = "static-nodes.json"  // Path within the datadir to the static node list
 	datadirTrustedNodes    = "trusted-nodes.json" // Path within the datadir to the trusted node list
 	datadirNodeDatabase    = "nodes"              // Path within the datadir to store the node infos
+
+	lowerCaseGeth = "geth"
+	upperCaseGeth = "Geth"
```

Comment:
Neither is this.

Reply:
is there a better way of addressing:
```
internal/build/env.go:66:26:warning: 2 other occurrence(s) of "True" found in: internal/build/env.go:66:61 internal/build/env.go:75:60 (goconst)
internal/build/env.go:66:61:warning: 2 other occurrence(s) of "True" found in: internal/build/env.go:66:26 internal/build/env.go:75:60 (goconst)
internal/build/env.go:75:60:warning: 2 other occurrence(s) of "True" found in: internal/build/env.go:66:26 internal/build/env.go:66:61 (goconst)
node/config.go:219:13:warning: 2 other occurrence(s) of "geth" found in: node/config.go:263:17 node/config.go:265:16 (goconst)
node/config.go:263:17:warning: 2 other occurrence(s) of "geth" found in: node/config.go:219:13 node/config.go:265:16 (goconst)
node/config.go:265:16:warning: 2 other occurrence(s) of "geth" found in: node/config.go:219:13 node/config.go:263:17 (goconst)
```
?

Reply:
could use `// nolint: goconst` but that sort of defeats the purpose IMO

Reply:
@karalabe may differ, but I feel goconst should not be enabled on CI because it reports things like this. We can enable `ineffassign`, `nakedret`, etc. because they're genuinely useful.

Reply:
Given that our code is generally clean, maybe I can agree with @fjl to keep this off. Perhaps reduce this PR to fixing the swarm repetition and it will be fine like that.

Reply:
I'm OK with adding `goconst` if we raise `--min-occurrences` to 6 or 7. There is value in using constants for repeated, but IMHO it's something that jumps in your face anyway while reviewing the code.

Reply:
good points; added the flag (which is suppoorted by metalinter itself) and reverted the code